docs/pages/access-controls/login-rules/reference.mdx

```diff
@@ -676,3 +676,11 @@ Expression | Result
 ---------- | ------
 `union(set("a"), set("b"))` | `("a", "b")`
 `union(set("a", "b"), set("b", "c"))` | `("a", "b", "c")`
+
+## Maintaining Login Rule objects
```

Comment:
This page is a reference for "the expression language that powers Login Rules" (per the intro paragraph), so I'm not sure this section is appropriate here.

Reply:
removed